src/service/transaction.service.ts

```diff
@@ -51,7 +51,12 @@ export class TransactionService {
             } else {
                 transactionFormatted += transaction.recipient.toHex();
             }
-            transactionFormatted += transaction.message.payload.length > 0 ? ' Message:\"' + transaction.message.payload + '\"' : '';
+            transactionFormatted += '';
+            if (transaction.message.payload != undefined){
```

Comment:
@rg911 Do you know of a better way to evaluate this in typescript? I don't like much using undefined.

Reply:
You shouldn't expecting `undefined here. Instead, if message is empty, SDK returns `EmtpyMessage` object.